pyartcd/tests/test_bump_rpms.py

```diff
@@ -0,0 +1,33 @@
+import unittest
```

Comment:
can we rename this file to `test_rebuild_golang_rpms.py` ? 
Then this file will be the place for all tests related to the logic in file `rebuild_golang_rpms.py`

Reply:
test file renamed